src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -120,6 +126,8 @@ internal static ErrorCode GetErrorCode(this LanguageVersion version)
                     return ErrorCode.ERR_FeatureNotAvailableInVersion7;
                 case LanguageVersion.CSharp7_1:
                     return ErrorCode.ERR_FeatureNotAvailableInVersion7_1;
+                case LanguageVersion.CSharp7_2:
+                    return ErrorCode.ERR_FeatureNotAvailableInVersion7_2;
```

Comment:
Thanks for the clarification.

I have a follow-up issue to use a generic diagnostic ID.
@AlekseyTs Any other feedback?